samples/memfs/inode.go

```diff
@@ -33,6 +33,9 @@ type inode struct {
 	// Mutable state
 	/////////////////////////
 
+	// Name of the inode
```

Comment:
Is this a relative name, as opposed to a path that can contain slashes? Maybe add an example.